docs/dev/uap.rst

```diff
@@ -0,0 +1,14 @@
+Users, Authentication, and Permissions
+======================================
+
+This is a core module found in ``kolibri/auth``.
+
+
+.. include:: uap/key_concepts.rst
+
+
+Models
+------
+
+.. automodule:: kolibri.auth.models
```

Comment:
Absolutely! Example of another use case: http://kolibri.readthedocs.org/en/latest/usage.html